cli/worker.rs

```diff
@@ -194,21 +194,6 @@ impl Worker {
     self.js_runtime.mod_evaluate(id)
   }
 
-  /// Loads, instantiates and executes provided source code
-  /// as module.
-  pub async fn execute_module_from_code(
-    &mut self,
-    module_specifier: &ModuleSpecifier,
-    code: String,
-  ) -> Result<(), AnyError> {
```

Comment:
I'd prefer to leave this method - I'm debugging test runner and there's a good chance this API will be needed.